src/main/java/org/arl/fjage/connectors/WebSocketConnector.java

```diff
@@ -171,8 +174,12 @@ public void run() {
           if (buf == null) break;
           s = new String(buf);
         }
-        for (WSHandler t: wsHandlers)
+        int i = 1;
+        int l = wsHandlers.size();
+        for (WSHandler t: wsHandlers) {
+          log.finest("Writing to "+ (i++) + " of "+l+" open sessions");
```

Comment:
I might recommend to remove this debug statement, as its in a heavily used loop and will have a performance impact even when the log level is disabled.

Reply:
And that should remove this whole hunk of change.

Reply:
OK. I'll pick this up in https://github.com/org-arl/fjage/pull/212